source/Engine.cpp

```diff
@@ -634,8 +634,16 @@ void Engine::Step(bool isActive)
 	info.SetString("date", player.GetDate().ToString());
 	if(flagship)
 	{
-		info.SetBar("fuel", flagship->Fuel(),
-			flagship->Attributes().Get("fuel capacity") * .01);
+		const double fuelCap = 2000.;
+		// If there is a lot of fuel, display a solid homogenous bar,
+		// instead of many notches in the bar.
+		if(flagship->Attributes().Get("fuel capacity") <= fuelCap)
+		{
+			info.SetBar("fuel", flagship->Fuel(),
+				    flagship->Attributes().Get("fuel capacity") * .01);
+		} else {
+			info.SetBar("fuel", flagship->Fuel());
+		}
```

Comment:
```suggestion
		const double MAX_FUEL_DISPLAY = 5000.;
		double fuelCap = flagship->Attributes().Get("fuel capacity");
		// If there is a lot of fuel, display a solid homogenous bar,
		// instead of many notches in the bar.
		if(fuelCap <= MAX_FUEL_DISPLAY )
			info.SetBar("fuel", flagship->Fuel(), fuelCap* .01);
		else
			info.SetBar("fuel", flagship->Fuel());
```
The MAX_FUEL_DISPLAY variable should then be moved to below RADAR_SCALE at the top of the file.